Design-Docs/Release-Policies.md

```diff
@@ -65,6 +65,18 @@ When features of these categories will change in non-backwards compatible ways o
 
 Features in categories 3 and 4 have no guarantees to be included in the next release, to not change in incompatible ways or to being promoted to categories 2 or 1. However, in our monthly newsletters, we will talk about these from time to time, so our partners can easily stay up to date in their understanding where we are going.
 
+## Backports
+
+A strong focus of SCS is on assuring the upstream projects are healthy and strive. For this to be possible a strong effort is made to assure features and fixes are implemented upstream.
+Sometimes the upstream release cycle will have an impact on when a feature or fix is available for SCS consumers. Backporting could then be considered an option.
+The following aspects should be considered in these case:
+
```

Comment:
Should we add ``Backports should be avoided whenever possible.`` as the first point here?

Reply:
I agree! This makes our preliminary standpoint clear in the very beginning.

Reply:
thanks! Very good point @berendt. added it.